core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
+  
+i.oppia-share-btn-fa {
+    display: inline-block;
```

Comment:
@markhalpin That would look cool. Shall I make separate classes for them? So as to include the color to specific icon